fails to find Qt6_DIR due to Unknown CMake command "_qt_internal_suggest_dependency_debugging"

--- a/tests/auto/CMakeLists.txt
+++ b/tests/auto/CMakeLists.txt
@@ -2,7 +2,6 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
 add_subdirectory(qsensor)
-add_subdirectory(cmake)
 if(TARGET Qt::Quick)
     add_subdirectory(qml)
 endif()
